docs/dev/pr.md

```diff
@@ -12,17 +12,22 @@ Message text should start from an upper case letter. If commit doesn't fix or im
 
 Possible tags:
 
-  * `[cpp]` - use, when you made changes in cpp wrapper code;
-  * `[memory]` - use, when you made changes in c core code;
-  * `[tests]` or `[test]` - use, when you made changes in tests;
-  * `[tools]` - use, when you made changes in tools;
-  * `[builder]` - use, when you made changes in builder tools;
-  * `[codegen]` - use, when you made changes in code generator;
-  * `[review]` - use, when you made commit with review fixes;
-  * `[refactor]` - use, when you made commit with some code refactoring;
-  * `[changelog]` - use, when you update changelog;
-  * `[docs]` - use, when you update documentation;
-  * `[python]` - use, when you made commit with python module changes.
+  * `[builder]` - changes in builder tool
+  * `[changelog]` - update changelog
+  * `[cmake]` - changes in cmake build system
+  * `[codegen]` - changes in code generator
+  * `[core]` - changes in `sc-storage` code
+  * `[doc]` or `[docs]` - update documentation
+  * `[git]` - changes in git configuration
+  * `[python]` - changes in python modules
+  * `[sc-kpm]` - changes in `sc-kpm` code
+  * `[sc-memory]` - changes in `sc-memory` code
+  * `[sc-server]` - changes in `sc-server` code
+  * `[sc-network]` - changes in `sc-network` code
+  * `[scs]` - changes in `scs` support
+  * `[test]` or `[tests]` - changes in tests
+  * `[thirdparty]` - updated thirdparty libraries
+  * `[web]` - changes in web interface
 
```

Comment:
We probably need to add `scripts` and `travis` tags to this list

Reply:
Maybe we will use common `ci` instead of `travis`?

Reply:
Agree, lets use `ci`